python/cudf/cudf/utils/_setup_numba.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+import glob
+import os
+
+from numba import config
+from numba.cuda.cudadrv.driver import Linker
```

Comment:
In `__init__`.py we say that `_setup_numba` needs to be executed before `from numba import cuda` in order to ensure that configuration is set before the import. Doesn't this import cause problems then because it will execute the contents of `numba/cuda/__init__.py`?

Reply:
Ah, this was true for the environment variable method in numba 0.57, but not true with the linker patching method. I'll update the comment.